### DISCUSSION TITLE
Can I change my Domain Name on an existing Azuracast Install?

### DISCUSSION THREAD

**Discussion by pwrj** [AzuraCast/AzuraCast#4082] [Q&A]
I want to change my domain name on Azuracast? Is that possible, if so how?

Docker Installation Method.

**Comment by renatopinto89** [comment]
If you have a domain, you can manage domains and subdomains on CloudFlare for free

Just forward your domain to your servers ip address and you're all set.
Avoid using the CloudFlare proxy tho. That tends to be problematic with ShoutCast/IceCast. (when you're forwarding the IP to domain click the cloud and select no proxy)

**Comment by pwrj** [comment]
Thanks. So If I wanna replace an old domain, just add the domain?

**Comment by Liam024** [comment]
`So If I wanna replace an old domain, just add the domain?`

@pwrj Yes.
